src/include/npkit/npkit.h

```diff
@@ -14,6 +14,12 @@
 #include "npkit/npkit_event.h"
 #include "npkit/npkit_struct.h"
 
+#if defined(__gfx1100__) || defined(__gfx1101__) || defined(__gfx1102__)
```

Comment:
since __builtin_amdgcn_s_memrealtime is available in gfx9 and gfx10, we can use following to be future proof:
#ifdef __GFX9__ || __GFX10__
#define NPKIT_GET_GPU_TIMESTAMP __builtin_amdgcn_s_memrealtime
#else
#define NPKIT_GET_GPU_TIMESTAMP clock64
#endif

Reply:
Fixed.